catalog/admin/admin-poller-service/src/main/java/org/codice/ddf/catalog/admin/poller/AdminPollerServiceBean.java

```diff
@@ -107,7 +107,7 @@ public AdminPollerServiceBean(ConfigurationAdmin configurationAdmin) {
         try {
             objName = new ObjectName(AdminPollerServiceBean.class.getName() + SERVICE_NAME);
         } catch (MalformedObjectNameException e) {
-            LOGGER.error("Unable to create Admin Source Poller Service MBean with name [{}].",
+            LOGGER.info("Unable to create Admin Source Poller Service MBean with name [{}].",
```

Comment:
We need to add integration tests to make sure all of the mbeans come up and work.  I am afraid we could ship with a broken mbean since this is now at info.